linera-execution/src/execution.rs

```diff
@@ -214,75 +214,6 @@ where
         Ok(())
     }
 
-    /// Schedules application registration messages when needed.
-    ///
-    /// Ensures that the outgoing messages in `results` are preceded by a system message that
-    /// registers the application that will handle the messages.
-    pub async fn update_execution_outcomes_with_app_registrations(
```

Comment:
Great! This will also fix https://github.com/linera-io/linera-protocol/issues/2362. :tada: